Catappult/appcoins-wallet-android
# [feature/APPC-2266] Perks Description Update

**What does this PR do?**

   This PR adds support to the latest API of user_stats where now there's two different descriptions: one for notifications, and other for the rest (perks). 

**Database changed?**

   Yes. Note that the old description was removed and new two description fields were added. I decided to copy the value of the old description to these new two fields on migration, but I'm open for better ways.

**How should this be manually tested?**

  Check if the description on notifications and on the promotions page is different (it should be). Also test migration and check if everything is ok. Note that the description don't match the ones on Zeplin, that will eventually be changed on the backend.

**What are the relevant tickets?**

  [APPC-2266](https://aptoide.atlassian.net/browse/APPC-2266)

**Code Review Checklist**
